### PR TITLE
oVirt e2e: add ovirt missing jobs

### PR DESCRIPTION
**Pull request by Gal-Zaidman**
Adding:

- openshift-cluster-api-provider-ovirt-release-{4.5,4.6,4.7}
- openshift-installer-release-4.5-presubmits

Signed-off-by: Gal-Zaidman <gzaidman@redhat.com>

### CODE REVIEW

**Comment by Gal-Zaidman** [comment]
/hold

Will need rebase on https://github.com/openshift/release/pull/9579

File: ci-operator/jobs/openshift/installer/openshift-installer-release-4.5-presubmits.yaml [hunk 1]
```diff
@@ -989,6 +989,85 @@ presubmits:
...
+    context: ci/prow/e2e-ovirt
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-installer-release-4.5-e2e-ovirt
+    optional: true
+    rerun_command: /test e2e-ovirt
+    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
```
Comment (sdodson) [inline ci-operator/jobs/openshift/installer/openshift-installer-release-4.5-presubmits.yaml:1006]: We should start leveraging run_if_changed more effectively in these jobs. Don't need to do that now though.

**Comment by sdodson** [comment]
> openshift-installer-release-4.5-presubmits

It's already in 4.6 and later or is there a reason we're not adding those as well?
/lgtm
/hold

**Comment by Gal-Zaidman** [comment]
> > openshift-installer-release-4.5-presubmits
> 
> It's already in 4.6 and later or is there a reason we're not adding those as well?
> /lgtm
> /hold

its in 4.6 and 4.7:

https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/installer/openshift-installer-release-4.6-presubmits.yaml#L1881

https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/installer/openshift-installer-release-4.7-presubmits.yaml#L1881

**Pull request merged**